src/translations/en.json

```diff
@@ -3899,43 +3899,32 @@
         "ram_usage": "Core RAM Usage"
       }
     },
-    "snapshot": {
```

Comment:
We should rename these keys in Localize before we merge this PR so we don't lose translations.

Reply:
No all, those containing "snapshot" should be removed.